libbeat/docs/shared-autodiscover.asciidoc

```diff
@@ -15,6 +15,10 @@ Autodiscover providers work by watching for events on the system and translating
 events with a common format. When you configure the provider, you can use fields from the autodiscover event to set
 conditions that, when met, launch specific configurations.
 
+On start, {beatname_uc} will scan all the already existing containers and launch the proper configs for them. Then
```

Comment:
I think you can probably just say "scan existing containers" (the phrase "all the already existing containers" seems redundant).